exception - cannot make a new request because the previous response is still open: please call response.close()

```
org.jetbrains.teamcity.rest.TeamCityConversationException: Failed to connect to https://@@@@@: cannot make a new request because the previous response is still open: please call response.close()
	at org.jetbrains.teamcity.rest.TeamCityInstanceImpl$service$3.handleError(implementation.kt:155)
	at org.jetbrains.teamcity.rest.TeamCityInstanceImpl$service$3.handleError(implementation.kt:104)
	at retrofit.RestAdapter$RestHandler.invoke(RestAdapter.java:242)
	at com.sun.proxy.$Proxy2.artifactContent(Unknown Source)
	at org.jetbrains.teamcity.rest.BuildImpl.openArtifactInputStreamImpl(implementation.kt:1366)
	at org.jetbrains.teamcity.rest.BuildImpl.openArtifactInputStream(implementation.kt:1338)
	at org.jetbrains.teamcity.rest.BuildArtifactImpl.openArtifactInputStream(implementation.kt:1602)
	at org.jetbrains.intellij.sharedIndexes.feed.Teamcity_inputKt.downloadFromTeamCity(teamcity-input.kt:46)
```

```
Caused by: retrofit.RetrofitError: cannot make a new request because the previous response is still open: please call response.close()
 	at retrofit.RetrofitError.unexpectedError(RetrofitError.java:44)
 	at retrofit.RestAdapter$RestHandler.invokeRequest(RestAdapter.java:400)
 	at retrofit.RestAdapter$RestHandler.invoke(RestAdapter.java:240)
 	... 46 more
 Caused by: java.lang.IllegalStateException: cannot make a new request because the previous response is still open: please call response.close()
 	at okhttp3.internal.connection.RealCall.enterNetworkInterceptorExchange(RealCall.kt:223)
 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:66)
 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:100)
 	at org.jetbrains.teamcity.rest.RetryInterceptor.intercept(implementation.kt:75)
 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:100)
 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:197)
 	at okhttp3.internal.connection.RealCall.execute(RealCall.kt:148)
 	at com.jakewharton.retrofit.Ok3Client.execute(Ok3Client.java:41)
 	at retrofit.RestAdapter$RestHandler.invokeRequest(RestAdapter.java:326)
```

--- a/teamcity-rest-client-impl/src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt
+++ b/teamcity-rest-client-impl/src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt
@@ -71,6 +71,7 @@ private class RetryInterceptor : Interceptor {
         while (response.retryRequired() && tryCount < 3) {
             tryCount++
             LOG.warn("Request ${request.url} is not successful, $tryCount sec waiting [$tryCount retry]")
+            runCatching { response.close() }
             Thread.sleep((tryCount * 1000).toLong())
             response = chain.proceed(request)
         }

